Coding/fw/lib/Lawicel/Lawicel.h

```diff
@@ -0,0 +1,157 @@
+    };
+
+    uint8_t charToByte(char MSB, char LSB); /**< Translates char symbols of a Byte into hex values */
+    uint8_t charToInt(char symbol);         /**< Translates char symbols of numbers into int values */
+    uint32_t IdDecode(bool extended);       /**< Translates char ID into value */
+
+    uint8_t receiveCommand();          /**< Receives and Interprets Buffer with Serial Command */
+    uint8_t CMD_Set_Baudrate();        /**< Sets Baudrate through presets */
+    uint8_t CMD_Set_BTR();             /**< Sets Baudrate through Registers */
+    uint8_t CMD_Open_Normal();         /**< Opens CAN Channel in Normal Mode */
+    uint8_t CMD_Open_Listen_Only();    /**< Opens CAN Channel in Listen-Only Mode */
+    uint8_t CMD_Close();               /**< Closes CAN Channel */
+    uint8_t CMD_Tx_Std();              /**< Transmits standard CAN Frame (11-bit ID) */
+    uint8_t CMD_Tx_Ext();              /**< Transmits extended CAN Frame (29-bit ID) */
+    uint8_t CMD_Tx_Std_RTR();          /**< Transmits standard RTR CAN Frame (11-bit ID) */
+    uint8_t CMD_Tx_Ext_RTR();          /**< Transmits extended RTR CAN Frame (29-bit ID) */
+    uint8_t CMD_Poll_Single();         /**< Poll incomming FIFO for CAN frames (single poll) */
+    uint8_t CMD_Poll_All();            /**< Polls incomming FIFO for CAN frames (all pending frames) */
+    uint8_t CMD_Poll_Auto();           /**< Toggles Auto Poll for inconming Frames */
+    uint8_t CMD_Flags();               /**< Read Status Flags */
+    uint8_t CMD_Set_Filter_Mode();     /**< Sets Filter Mode 0 = Dual-Filter, 1 = Single-Filter */
+    uint8_t CMD_Set_ACn();             /**< Sets Acceptance Code Register */
+    uint8_t CMD_Set_AMn();             /**< Sets Acceptance Mask Register */
+    uint8_t CMD_Set_Serial_Baudrate(); /**< Sets UART Baudrate (and saves setting on EEPROM) */
+    uint8_t CMD_Version();             /**< Sends Hardware and Software Version */
+    uint8_t CMD_Serial_Number();       /**< Sends Serial Number of Hardware */
+    uint8_t CMD_Timestamp();           /**< Toggles Timestamp (and saves setting on EEPROM) */
+    uint8_t CMD_Auto_Start();          /**< Auto Startup feature (from power on) */
+
+    uint8_t Autopoll(); /**< Frame Polling without any extra tags */
+    uint32_t getTimestamp(); /**< Returns Timestamp */
+
+    bool m_timestamp = false;   /**< Toggle timestamp */
+    bool m_autoPolling = true;  /**< Toggle Auto-Polling */
+    String m_serialInput;       /**< Input String for Serial-Message */
+    String m_serialReturn = ""; /**< String sent back to SerialAdapter */
+    uint8_t m_length = 0;       /**< Length of Serial-Message */
+    uint8_t m_autostart = 0;    /**< Sets Auto Start Mode */
+
+    SerialInterface *m_selectedSerial; /**< Active Serial Adapter */
+    CANInterface *m_selectedCAN;       /**< Active CAN Adapter */
+    NVMInterface *m_selectedNVM;       /**< Active NVM Adapter */
+
+    /**
```

Comment:
I would add another private: here, also it is already in a private section. This makes it more explizit. There is also a risk that someone addes a public: some lines above later. This would also make these public.